tests/testVue.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 
-i= basename "$PWD"
+mkcdir ()
+{
+    mkdir -p -- "$1" &&
+      cd -P -- "$1"
```

Comment:
Changing directories during the test command is not great, as when the tests failed the user will be confused as to why their terminal is suddenly in a new directory compared to when they started it. Instead, let's use pnpm `--dir` flag:

https://pnpm.io/pnpm-cli#-c-path---dir-path

Reply:
While we can use `--dir` for later executions once the application is created this won't apply to vue cli.
So I'm not sure if that will improve things a lot.

`vue create --help`
Does not list a similar function like pnpm --dir

Reply:
In that case, it would be best to try to catch the error and return the user back to their original working directory, if possible.

Reply:
It should do that by default since concurrently creates sub Processes right? The Process the Users Terminal is running in won't change.